src/wallet/rpc/wallet.cpp

```diff
@@ -1156,7 +1157,8 @@ Span<const CRPCCommand> GetWalletRPCCommands()
         {"wallet", &sendtoaddress},
         {"wallet", &sethdseed},
         {"wallet", &setlabel},
-        {"wallet", &settxfee},
+        {"hidden", &settxfee},
```

Comment:
Do we set deprecated rpc methods to hidden? I'm not sure I have seen that before.

Reply:
it's what it's suggested on https://github.com/bitcoin/bitcoin/issues/31088

Reply:
Looking back it seems like we haven't hidden them, just deprecated. That's also what I would suggest here.

Reply:
Do you have an example of deprecated RPC so I can compare on how it was done before?

Reply:
Checking a few of these should give you an overview: https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+Deprecate+in%3Atitle+rpc+in%3Atitle+is%3Aclosed+

Reply:
Thanks!
Applied following some of the examples.